enhancements/installer/cluster-api-based-installations.md

```diff
@@ -0,0 +1,485 @@
+---
+title: cluster-api-based-installations
+authors:
+  - "@patrickdillon"
+  - "@vincepri"
+  - "@JoelSpeed"
+reviewers:
+  - "@sdodson"
+  - "@2uasimojo"
+  - "@mhrivnak"
+  - "@jnpacker"
+  - "@dtantsur"
+  - "@mdbooth"
+  - "@rvanderp3"
+  - "@jcpowermac"
+  - "@enxebre"
+  - "@wking"
+  - "@dhellmann"
+  - "@avishayt"
+approvers:
+  - "@zaneb"
+api-approvers:
+  - "None"
+creation-date: 2023-12-12
+last-updated: 2023-12-12
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/CORS-2840
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Enabling Cluster-API-based Installations via openshift-install
```

Comment:
Wow, yes, that was not at all clear when I read this. I suggest a new title, as well as mentioning explicitly what parts of CAPI are in/out of scope in the goals section.

Reply:
The entire Motivation section is about CAPI alignment and finding something to replace terraform. So I think it makes sense to have a title that reflects that by mentioning CAPI. Refining it to give better context is a good idea.

The "local control planes" part seems like an implementation detail, and it's even an undesirable one that we're trying to work around just because it's the price of admission for using CAPI.

Reply:
Going to add a note explaining the first phase limitations of this proposal

Reply:
Agree with Michael. The core of the idea here is using CAPI Infra providers in place of Terraform providers. So it's hard not to mention CAPI. But the current title is definitely misleading.

Reply:
"Bootstrap OpenShift control plane with CAPI Infra Providers"?

Reply:
Including `control plane` could be misconstrued with the CAPI concept of a control plane provider, which we are also looking to introduce.
What about `Bootstrap OpenShift cluster infrastructure with CAPI Infra Providers`?

Reply:
renamed accordingly